addons/filesystem/functions/fnc_initFilesystem.sqf

```diff
@@ -8,30 +8,58 @@
  * None
  */
 
-params['_entity'];
+params["_entity"];
 
 if(!isServer) exitWith {};
 
-private _filesystem = [createHashMapFromArray [
+private _filesystem = [createHashMapFromArray [], 'root', [[true, true, true], [true, true, false]]];
 
-	// Essential user commands
-	["bin", [createHashMap, 'root', [[true, true, true], [true, true, false]]]],
+/* ================================================================================ */
 
-	// Essential system commands
-	["sbin", [createHashMap, 'root', [[true, true, true], [true, true, false]]]],
+private _config = configFile >> "AE3_FilesystemObjects";
 
-	// User home dirs.
-	["home", [createHashMap, 'root', [[true, true, true], [true, true, false]]]],
+private _filesystemObjects = ("inheritsFrom _x == (configFile >> 'AE3_FilesystemObject')" configClasses _config);
 
-	// Root home dir
-	["root", [createHashMap, 'root', [[true, true, true], [false, false, false]]]],
+{
+	private _ptr = [];
 
-	// Mount file for tmp filesystems
-	["mnt", [createHashMap, 'root', [[true, true, true], [true, true, false]]]],
+	try
```

Comment:
Why try here?

Reply:
Perhaps a testing artefact. I will remove that.